wordpress-mobile/WordPress-iOS
# Fixes media library sharing issues.

Resolves an issue that prevented users from sharing images and videos out of our media library.

This was reported by @yaelirub a while back, but it proved tricky to resolve and it took me quite a while.

Fyi @startuptester - Apologies for the delay but this one was tricky.

## To test:

1. Go to the media library for a site.
2. Tap on an image of video.
3. Tap the sharing icon (sharing arrow).
4. Make sure you can share images and videos.

You can test the same in develop to see it was not working properly.

## Regression Notes

1. Potential unintended areas of impact

Sharing images and videos from the media library, although this PR was created to fix the issues we already had.
Also @bjtitus might be able to mention if there's a chance this may affect other parts of the App.

2. What I did to test those areas of impact (or what existing automated tests I relied on)

This fixes a broken flow so the testing steps should cover it.  For the rest @bjtitus may be able to mention if there's a risk.

3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
